ecs-cli/modules/commands/flags.go

```diff
@@ -63,9 +63,10 @@ const (
 	UntaggedFlag   = "untagged"
 
 	// Compose
-	ProjectNameFlag     = "project-name"
-	ComposeFileNameFlag = "file"
-	TaskRoleArnFlag     = "task-role-arn"
+	ProjectNameFlag       = "project-name"
+	ComposeFileNameFlag   = "file"
+	TaskRoleArnFlag       = "task-role-arn"
+	ECSFieldsFileNameFlag = "ecs-fields"
```

Comment:
I like calling the flag "ecs-fields", its short, concise, and the meaning is clear.